src/hr/algebra/pi/model/Employee.java

```diff
@@ -1,23 +1,13 @@
 package hr.algebra.pi.model;
 
 public class Employee {
```

Comment:
Hint: Maybe Employee should be responsible for parking and refilling vehicles?

Reply:
Agree, i will fix that.